massa-execution-worker/src/execution.rs

```diff
@@ -901,7 +828,12 @@ impl ExecutionState {
             .active_cursor
             .get_next_slot(self.config.thread_count)
```

Comment:
can you verify that we should be getting the next slot and not just active_cursor directly ?

Reply:
See https://github.com/massalabs/massa/pull/2701#discussion_r901531034